lib/gcloud.rb

```diff
@@ -374,4 +374,41 @@ def logging scope: nil
     require "gcloud/logging"
     Gcloud.logging @project, @keyfile, scope: scope
   end
+
+  ##
+  # Creates a new object for connecting to the Translate service.
+  # Each call creates a new connection.
+  #
+  # TODO: Add info for creatign an API Key here...
+  # TODO: Explain that the API Key is likely temproary and can change in a
+  # future release.
+  #
+  # @param [String] key API Key is blah blah blah...
+  #
+  # @return [Gcloud::Translate::Api]
+  #
+  # @example
+  #   require "gcloud"
+  #
+  #   gcloud = Gcloud.new
+  #   translate = gcloud.translate "api-key-abc123XYZ789"
+  #
+  #   translation = translate.translate "Hello world!", to: "la"
+  #   puts translation #=> Salve mundi!
```

Comment:
No, that's what it prints. When you print a string it doesn't also print the quotes. If you inspect the object it would show this:

```ruby
#<Gcloud::Translate::Translation:0x007fe831e0b5e8 @text="Salve mundi!", @to="la", @origin="Hello world!", @from="en", @detected=true>
```

But you can print it like a string because the `Translation` object has `#to_s` and `#to_str`.